packages/vm/benchmarks/bytecode.ts

```diff
@@ -0,0 +1,93 @@
+import { readFileSync } from 'fs'
+import Benchmark from 'benchmark'
```

Comment:
Curious if there is a reason to use this benchmark package rather than vitest built in benchmarking features?

Reply:
I haven't had a chance to review the whole PR yet but our use of `benchmark` long predates our use of `vitest`. We don't use benchmarking of this sort all that much in our current development process but there may be a reason to switch at some point.

Reply:
It's consistent with the existing implementation, see `run.ts` and `mainnetBlocks.ts`. I don't mind migrating to  `vitest` if that's the preferred way.

Reply:
@JacekGlen how much work would it be to switch all of this to the `vitest` benchmarking tools? Looks like it produces some [nice output](https://vitest.dev/guide/features.html#benchmarking) and might be easier to use over all.

Reply:
If helpful, I am doing vitest benchmark testing in the [test/bench](https://github.com/evmts/tevm-monorepo/tree/main/test/bench) folder

Reply:
@acolytec3 @roninjin10 I have updated the benchmarks to use `tinybench` (via `vitest`). 
The reason for using the benchmark library directly, rather than unit-test style preferred by `vitest`, is that this is more command-line tool. Using the benchmark library directly gives better control over parameters and output.